### PR TITLE
Added Applied Economics Minor

### PR DESCRIPTION
**Pull request by mirandayu131**
**Summary**
Added the Business minor for Applied Economics in the SC Johnson College of Business. 

_Information on the requirements is sourced from the following:_ 
[https://business.cornell.edu/programs/undergraduate/minors/applied-economics/]

**Test Plan**
Test AE minor!

### CODE REVIEW

**Comment by zachary0kent** [comment]
@jerrry1123 @rohanmaheshwari430 might want to review this for dev portfolio.

**Comment by noschiff** [comment]
make sure the minor is added to the req JSON!

File: src/data/index.ts [hunk 1]
```diff
@@ -375,6 +376,12 @@ const json: RequirementsJson = {
     },
   },
   minor: {
+    AEM: {
+      name: 'Applied Economics and Management',
+      schools: ['AG', 'BU'],
```
Comment (noschiff) [inline src/data/index.ts:381]: shouldn't every college except dyson be allowed to do the minor?

**Comment by zachary0kent** [comment]
Remember to run `npm run req-gen`!

**Review by noschiff — CHANGES_REQUESTED** [review]
Can we name this "Applied Economics" instead of AEM? The [minor](https://business.cornell.edu/programs/undergraduate/minors/applied-economics/) is officially applied economics, whereas AEM usually refers to the Dyson major (M for & Management).

File: src/data/index.ts (hunk 1 above)
Comment (noschiff) [reply src/data/index.ts:381]: @zachary0kent am I wrong about how this works

**Comment by noschiff** [comment]
@mirandayu131 can you add more information to the PR overview? It'd be good to have a link to the source of requirements for the reviewer to reference, as well as a test plan. The test plan for adding a major is usually pretty brief though :)

**Review by noschiff — COMMENTED** [review]
Looks great! I'll leave the approval + testing to one of the devs for their portfolio :)

File: src/data/index.ts (hunk 1 above)
Comment (jerrry1123) [reply src/data/index.ts:381]: i know that some colleges like engineering ahve their own version of their AEM minor

**Comment by jerrry1123** [comment]
I did the testing plan and all the requirements look right, I just think that it might be better to rename the minor to non-engineering or something like that to differentiate this minor from the engineering business minor.

**Review by jerrry1123 — APPROVED** [review]
I did the testing plan and all the requirements look right, I just think that it might be better to rename the minor to non-engineering or something like that to differentiate this minor from the engineering business minor.

File: src/data/index.ts (hunk 1 above)
Comment (zachary0kent) [reply src/data/index.ts:381]: I believe this refers to which college offers the minor

Comment (noschiff) [reply src/data/index.ts:381]: where do we store which colleges are eligible? @zachary0kent

Comment (zachary0kent) [reply src/data/index.ts:381]: I don't believe we do? I'm not of such restrictions typically existing anyways.

**Pull request merged**